conf/modules.config

```diff
@@ -22,6 +22,31 @@ process {
         ext.args = '--quiet'
     }
 
+    withName: 'KRAKEN2_KRAKEN2' {
+        publishDir = [
```

Comment:
I think there is a general statement for this in this (file)[https://github.com/nf-core/seqinspector/blob/31c1f829d97c4b98d21b68beed4af050fd331a37/conf/modules.config#L15], so I don't think is needed to add it twice, except if that bit is going to be removed it later?

Reply:
There is! I just added it here for two reasons: the first is that I wanted more descriptive names for the folders (`kraken2_reports` instead of just `kraken2`) and I wanted the krona plots to be inside the `kraken2_reports` folder, with a more descriptive name as well.
The second reason I have added this seemingly redundant code is that kraken2 and kronatools can produce more output than what is produced now. I left these lines here looking into the future: they might need to be modified depending on the needs of the pipeline once it reaches a more stable status.